frontend/src/app/components/negotium-task/negotium-task-details/negotium-task-details.component.scss

```diff
@@ -0,0 +1,119 @@
+$green: #32f9a5;
+$input-grey: rgba(112, 112, 112, 0.11);
+
+$pink: #ef476f;
+$dark-blue: #073b4c;
+
+.title {
+  font-size: 2.5em;
+  text-align: center;
+  color: $green;
+  align-items: center;
+  display: flex;
+  padding: 0.5em;
+  font-family: 'Segoe UI', sans-serif;
+}
+
+.header {
+  border: 1px $input-grey solid;
+  margin: 0px 0px 40px 0px;
+}
+
+// .task-details {
+//   display: flex;
+//   flex-direction: column;
+//   justify-content: center;
+//   align-items: center;
+//   // border: 1px $input-grey solid;
+//   width: 25%;
+//   margin: 0px 40px 40px;
+//   padding: 2em;
+// }
+
+.tasks-paragraphs {
+  display: flex;
+  flex-direction: column;
```

Comment:
.task-paragraphs i control buttons mają wspólne style (jest jeszcze parę innych takich przykładów), może lepszym pomysłem będzie po prostu stworzenie klasy, która je zawiera?

Reply:
no pewnie, nie czyscilem tego kodu jeszcze do konca - mam to w planach :D